app/routes/yara_rules.py

```diff
@@ -562,3 +567,101 @@ def delete_all_inactive_yara_rules():
     db.session.query(yara_rule.Yara_rule).filter(yara_rule.Yara_rule.active == 0).delete()
     db.session.commit()
     return jsonify(''), 200
+
+
+@app.route('/ThreatKB/yara_rules/<int:yara_rule_id>/revert-to-revision/<int:revision>', methods=['PUT'])
```

Comment:
Decided to go with this as the endpoint.... and PUT as I'm updating the resource.

Let me know if you want a different endpoint, where the information is included in payload.